optaplanner-examples/src/main/java/org/optaplanner/examples/nqueens/domain/Queen.java

```diff
@@ -32,6 +32,16 @@
     // Planning variables: changes during planning, between score calculations.
     private Row row;
 
+    public Queen() {
+        // For XStream
```

Comment:
If we add that comment, we should do it in the other examples that have a default constructor for this reason too.
Personally, I wouldn't add the comment.

Btw, the comment isn't accurate. OptaPlanner itself also needs a no-arg constructor to planning clone Queen.